cloudstate/version.py

```diff
@@ -0,0 +1,8 @@
+
+# -*- coding: utf-8 -*-
+
+"""
+Licensed under the Apache License, Version 2.0.
+"""
+
+__version__ = "0.5.0"
```

Comment:
milestone is 0.1.0. Do you like to start with 0.5.x? So far I did not have, as an exmaple the Go support, aligned to the main project regarding the version numbers, they will probably diverge anyway.

Reply:
You the boss. I adopted the 0.5.x series in the libraries I support for compatibility with the active version of Cloudstate

Reply:
>You the boss.
I'm not :-)

I'm fine with 0.5.0, if we get aligned with Cloudstates version for this repo I'm fine with that. If we agree to have support libraries get aligned with major.minor versions of the main project I'm fine with that too.